consumer/start_consumer2.sh

```diff
@@ -0,0 +1,313 @@
+#!/bin/bash
```

Comment:
Do you intend to merge these shell scripts into the repo? If so they should probably be in a more descriptive directory name such as `scripts`

Reply:
This one could be removed - consumer2 was for hermes 1.12.0 - and consumer.sh is for hermes 0.15.0

Reply:
I think these scripts would be helpful for setting up local environment for future development of ICS.